D/dist_m4ri/build_tarballs.jl

```diff
@@ -0,0 +1,25 @@
+using BinaryBuilder, Pkg
+
+name = "dist_m4ri"
+version = v"0.0.1"
+
+sources = [
+    GitSource("https://github.com/QEC-pages/dist-m4ri.git",
+              "0c540cb766f5538d9a89152dc875ae5a357ee6c6")
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/dist-m4ri/src/
+make -j${nproc} CC=$CC CXX=$CXX
+install -Dvm 755 "dist-m4ri${exeext}"  -t "${bindir}"
+"""
+
+platforms = filter(p -> Sys.islinux(p) || Sys.isapple(p) || Sys.isfreebsd(p) || Sys.iswindows(p), supported_platforms())
```

Comment:
@Fe-r-oz , while Mose is the one to really say how these things are set up, I usually try with all `supported_platforms()` directly, without filtering anything out. That way you can even detect issues with the build recipe that have not surfaced in previous builds.